spec/Candid.md

```diff
@@ -1,6 +1,6 @@
 # Candid Specification
 
-Version: 0.1.1
+Version: 0.2
```

Comment:
I think the change is an extension of the existing rules. If we follow semantic versioning, it should be `0.1.2`?

Reply:
Not really: It’s an extension if a service starts relying on these rules, but not if a client starts to rely on them (and starts omitting optional fields). But I don't care as long as we have a new version, so that we can say things like “has motoko already updated the decoder to Candid 0.1.2”. We can use animal names instead, FWIW.

```suggestion
Version: 0.1.2
```